cloudnet/src/main/java/de/dytanic/cloudnet/util/VisualFileUtils.java

```diff
@@ -0,0 +1,112 @@
+package de.dytanic.cloudnet.util;
+
+import de.dytanic.cloudnet.console.IConsole;
+import de.dytanic.cloudnet.console.animation.ProgressBarInputStream;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.net.URLConnection;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+public class VisualFileUtils {
```

Comment:
This whole class is redundant, it's features are not really needed and it supports a bad code design